hack/update-codegen.sh

```diff
@@ -20,6 +20,8 @@ set -o pipefail
 
 KUBEEDGE_ROOT=$(unset CDPATH && cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd -P)
 
+export GOPATH="${GOPATH:-$(go env GOPATH)}"
+
```

Comment:
deepcopy-gen tool and others in https://github.com/kubeedge/kubeedge/blob/master/hack/generate-groups.sh need GOPATH env variable, by official doc
`--output-base string                   Output base; defaults to $GOPATH/src/ or ./ if $GOPATH is not set. (default "./")`
So now we must put the kubeedge codes under $GOPATH/src/github.com/kubeedge/kubeedge dir, or the generated codes will be output to another dir.